make/wag.mk

```diff
@@ -23,10 +23,17 @@ bin/wag: ensure-wag-version-set
 	@mkdir -p bin
 	$(eval WAG_VERSION := $(if $(filter latest,$(WAG_VERSION)),$(WAG_LATEST),$(WAG_VERSION)))
 	@echo "Checking for wag updates..."
-	@echo "Using wag version $(WAG_VERSION)"
-	@[[ "$(WAG_VERSION)" != "$(WAG_INSTALLED)" ]] && echo "Updating wag..." && curl --retry 5 -f -sL https://github.com/Clever/wag/releases/download/$(WAG_VERSION)/wag-$(WAG_VERSION)-$(SYSTEM)-amd64.tar.gz | tar -xz -C bin || true
-	@[[ "$(WAG_VERSION)" != "$(WAG_INSTALLED)" ]] && touch swagger.yml || true
+	@echo "Using wag version $(WAG_INSTALLED)"
+	@[[ "$(WAG_VERSION)" != "$(WAG_INSTALLED)" ]] && echo "Updating wag...to $(WAG_LATEST)"  && wget -P bin https://github.com/Clever/wag/releases/download/$(WAG_VERSION)/wag-$(WAG_VERSION)-$(SYSTEM)-amd64.tar.gz
```

Comment:
Shell commands confuse me a bit so i might be misunderstanding something - arent we `wget`ing `WAG_VERSION` and not necessarily `WAG_LATEST`? But the comment says we are updating to `WAG_LATEST`

Reply:
So yeah, sorry the verbiage change was last minute and I forgot that sometimes we want to download something that ISNT the latest and hence that's why it downloads WAG_VERSION. Changed the comment.